docs/src/test-cli-js.md

```diff
@@ -14,6 +14,11 @@ Here are the most common options available in the command line.
   npx playwright test
   ```
 
+- Run all the tests against a specific [project](./test-configuration-js.md#multiple-browsers)
```

Comment:
Let's move it below, where --browser options are. You can also remove --browser versions while you are here.